Tests/KlaviyoSwiftTests/StateManagementTests.swift

```diff
@@ -501,7 +509,7 @@ class StateManagementTests: XCTestCase {
         let initialState = INITILIZING_TEST_STATE()
         let store = TestStore(initialState: initialState, reducer: KlaviyoReducer())
 
-        let event = Event(name: .OpenedPush)
+        let event = Event(name: .OpenedAppMetric)
```

Comment:
change to non opened push event since it doesn't matter for this test which event we use.